actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -419,14 +419,14 @@ def cookies
     end
 
   private
-    ContentTypeHeader = Struct.new :mime_type, :charset
-    NullContentTypeHeader = ContentTypeHeader.new nil, nil
+    ContentTypeHeader = Struct.new :mime_type, :extra, :charset
+    NullContentTypeHeader = ContentTypeHeader.new nil, nil, nil
```

Comment:
`NullContentTypeHeader` is not used anywhere. Can that be removed? 

I tried removing it and running the spec everything worked fine for me. Just not sure if I should raise a PR or not.